src/bandersnatch/simple.py

```diff
@@ -23,6 +29,16 @@ class SimpleFormat(Enum):
     JSON = auto()
 
 
+class SimpleDigests(NamedTuple):
+    sha256: str
+    md5: str
```

Comment:
Not for this PR - but I wonder how we can support any hashname moving forward and just have the code adapt if PyPI change / offers new hashes and people want to move to it. I might open a Wishlist Issue.

I might in the short term add an exception / error message explaining we need a code diff for a new hash ...

Reply:
I think this should be doable if one creates the named tuple and enum with a functional approach rather than the static classes that were defined in this PR. Some decisions would definitely need to be made on how to check what algorithms are available.

Reply:
Totally, I feel we just pull what ever the config allows/asks for and error if it's not in the JSON metadata we pull. I might open a cleaerer issue / do it one day if I get time. I like to avoid static code that needs a code push to give people potential future functionality (e.g. a new hash to be supported) where ever possible.